docs/analytics_new_analysts/data-management.md

```diff
@@ -80,7 +80,7 @@ writer.save()
 ```
 
 ### Parquet
-Parquet is an "open source columnar storage format for use in data analysis systems." Columnar storage is more efficient as it is  easily compressed and the data is more homogenous. CSV files utilize a row-based storage format which is harder to compress, a reason why Parquets files are preferable for larger datasets. Parquet files are faster to read than CSVs, as they have a higher querying speed and preserve datatypes (i.e. Number,  Timestamps, Points). They are best for intermediate data storage and large datasets (1GB+) on most any on-disk storage. This file format is also good for passing dataframes between Python and R. A similar option is [feather](https://blog.rstudio.com/2016/03/29/feather/).
+Parquet is an "open source columnar storage format for use in data analysis systems." Columnar storage is more efficient as it is  easily compressed and the data is more homogeneous. CSV files utilize a row-based storage format which is harder to compress, a reason why Parquets files are preferable for larger datasets. Parquet files are faster to read than CSVs, as they have a higher querying speed and preserve datatypes (i.e. Number,  Timestamps, Points). They are best for intermediate data storage and large datasets (1GB+) on most any on-disk storage. This file format is also good for passing dataframes between Python and R. A similar option is [feather](https://blog.rstudio.com/2016/03/29/feather/).
```

Comment:
This is an accepted variant: https://grammarist.com/usage/homogenous-homogeneous/